pallets/investments/src/tests.rs

```diff
@@ -622,7 +622,7 @@ fn update_redeem_fails_when_collect_needed() {
 fn fulfillment_flow_for_everything_works() {
 	TestExternalitiesBuilder::build().execute_with(|| {
 		#[allow(non_snake_case)]
-		let PRICE: Rate = price_of(1, 2, 10);
+		let PRICE: Quantity = price_of(1, 2, 10);
```

Comment:
Why the change? In pallet pools we use `Rate`. It is confusing to change it here now IMO

Reply:
This price is not "related" to the one used in `pallet-pool-system`. Actually, the own `pallet-investment` defines such type as `Quantity` under `T::BalanceRatio` (namely price). We do not want 27 decimals for them, we want 18. Basically I forgot to modify it when I updated all rates some months ago.

The rule we are using is prices: `Quantity`; rates (used by interests where decimals are quite significant): `Rate`.

Reply:
Ahh, I see. Sorry, my bad you are right of course. I did overlook that the tranches are using `Quantity` too.